### ISSUE TITLE
Mac OSX Boards Not appearing, Search bar spawning

### ISSUE THREAD

**Issue by bryan-zake** [turtl/desktop#22]
This started for me a coupled days ago. It doesn't appear on my Windows 10 partition or on my android phone.
When I go to view my boards, none of them appear, only the "Search for boards" bar appears.
When I move back to the notes section from boards, the boards search bar doesn't disappear. Then if I go back to boards, another one pops up. And so on, until there are many search for boards bars on my screen.
<img width="735" alt="screen shot 2016-02-08 at 1 28 36 pm" src="https://cloud.githubusercontent.com/assets/13228821/12894928/d615856c-ce67-11e5-8ec6-a9df7408cc6e.png">
Restarting and re-downloading do not solve the issue. Though, restarting does get rid of the multiple search bars. xD
turtl version 41.0.2272.76 (2272.76)

**Comment by orthecreedence** [comment]
Wow, sorry you're having troubles. Can you open the console (ctrl+shift+k) and take a screenshot? There is probably some kind of error keeping the board list from displaying properly.

**Comment by bryan-zake** [comment]
<img width="1048" alt="screen shot 2016-02-09 at 10 40 08 am" src="https://cloud.githubusercontent.com/assets/13228821/12921056/76d7ceb8-cf19-11e5-8dc5-f6815f5c5344.png">

**Comment by orthecreedence** [comment]
Thanks for the screenshot. Weird, it seems as though somehow a board got created without a title in your profile and the code is expecting one to always be there. I'll push out a fix for this (tracked https://trello.com/c/q0u878bg).

In the meantime if you open that console, type this, and hit enter/return:

```javascript
turtl.profile.get('boards').filter(function(b) { return !b.get('title'); }).forEach(function(b) { b.set({title: '(no title)'}).save() });
```

This will rename and save any board without a title. I'm hoping this will fix your problem in the meantime.

**Comment by orthecreedence** [comment]
Ok should be fixed in 92d6a6d, closing for now.
